specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+          },
+          {
+            "name": "blobDetails",
+            "description": "The details (name and container) of the blob to store or download data.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BlobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Get a SAS token to upload data for a Quantum Workspace": {
+            "$ref": "./examples/storageGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SasUriResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/quotas": {
+      "get": {
+        "operationId": "Quotas_List",
+        "description": "List quotas for the given workspace.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a list of quotas for a particular quantum workspace.": {
+            "$ref": "./examples/quotaList.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/QuotaList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "JobDetailsList": {
+      "description": "List of job details.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobDetails"
+          }
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total records count number."
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "JobDetails": {
+      "description": "Job details.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The job id."
```

Comment:
Is id eligible to be ready-only?

Reply:
Not sure, but I think out of scope for this PR since I didn't make changes here and don't have the service context about this.